### ISSUE TITLE
Separate inventory between different worlds

### ISSUE THREAD

**Issue by Peppe-73** [GeorgH93/Minepacks#304]
Hi, I'm trying your plugin, but I encountered a problem, I have 3 worlds on my server where the player can go between worlds, the inventory is separate, but your plugin takes precedence over the configuration and I end up with the backpack with the item in the inventory. Let me explain better, if I have a full backpack in my inventory from the Creative world, I must not have the same backpack in the Survival world full of items from the Creative world. I hope I explained myself well, can the problem be solved?

Basically the contents of the backpacks must remain in the worlds in which they were used. Right now if a player on my server goes to creative he can transport materials to the Survival world and this must not happen. I saw that there is a blacklist and whitelist in the CFG file, but it prevents the use of the backpack if I specify the worlds, but I want the backpack to be used, only the contents must be separated

**Comment by Peppe-73** [comment]
In the configuration file you should add an entry where you specify in which worlds inventory and unified. So automatically where I don't specify the name of the inventory world it is separated.
